seq2seq/third_party/utils/utils.py

```diff
@@ -1,3 +1,16 @@
+# Copyright 2020 The HuggingFace Team. All rights reserved.
+#
```

Comment:
it looks like we have made a very minor modification to this file. Is the modification necessary or will it work if we revert it? If it isn't necessary, please revert it.

If it is necessary, we should add the usual comment about how this file has been modified (as described in the doc).

Reply:
this is not modified, more than formatting and changing the imports which needs to be done, handled it as category 3.